inc/admin.php

```diff
@@ -248,3 +258,20 @@ function get_error_messages() {
 
 	return $messages;
 }
+
+/**
+ * Check whether there is an existing ads.txt file that will override the plugin output.
+ *
+ * @return bool Value of whether ads.txt already exists
+ */
+function ads_txt_exists() {
+	// No-op on WordPress.com
+	if ( defined( 'WPCOM_IS_VIP_ENV' ) && WPCOM_IS_VIP_ENV ) {
+		return;
+	}
+
+	$ads_txt_path = get_site_url() . '/ads.txt';
+
+	// Return true if file exists, otherwise return false
+	return 200 == wp_remote_get( $ads_txt_path )['response']['code'];
```

Comment:
```suggestion
	return 200 === wp_remote_get( $ads_txt_path )['response']['code'];
```

Reply:
Removed, since we're refactoring to be a JS-only check.